server/models/models1.js

```diff
@@ -1,4 +1,4 @@
-var db = require('../dbs/db1.js');
+var db = require('../dbs/db2.js');
```

Comment:
Looks like this file isn't in use, can it be deleted?

Reply:
Easily.